gateway-service/src/main/resources/application.yml

```diff
@@ -165,7 +165,9 @@ eureka:
                       gatewayUrl: api/v1
                       swaggerUrl: https://${apiml.service.hostname}:${apiml.service.port}/api-doc
                       documentationUrl: https://zowe.github.io/docs-site/
-
```

Comment:
Why do you replicate the information here?
Isn't it enough to use directly the apiml.security.auth.provider?